FBShimmering/FBShimmeringLayer.m

```diff
@@ -442,7 +453,11 @@ - (void)_updateShimmering
       slideAnimation = shimmer_slide_animation(self, animationDuration, _shimmeringDirection);
       slideAnimation.fillMode = kCAFillModeForwards;
       slideAnimation.removedOnCompletion = NO;
-      slideAnimation.beginTime = CACurrentMediaTime() + fadeOutAnimation.duration;
+      if (_shimmeringBeginTime != FBShimmerDefaultBeginTime) {
+        slideAnimation.beginTime = _shimmeringBeginTime;
+      } else {
+        slideAnimation.beginTime = CACurrentMediaTime() + fadeOutAnimation.duration;
```

Comment:
only comment: this should set _shimmeringBeginTime to the same value, so you can read it out from the shimmering layer